BUILD

```diff
@@ -71,6 +72,16 @@ kt_jvm_library(
     tags = ["maven_coordinates=com.vaticle.typedb:typedb-studio:{pom_version}"],
 )
 
+kt_compiler_plugin(
+    name = "org_jetbrains_compose_compiler_plugin",
+    id = "jetbrains.compose.compiler",
+    target_embedded_compiler = True,
+    visibility = ["//visibility:public"],
+    deps = [
+        "@maven//:org_jetbrains_compose_compiler_compiler",
+    ],
+)
+
```

Comment:
We define the compiler plugin here via the provided `rules_kotlin` method `kt_compiler_plugin`.